src/bin/common/solving_loop.ml

```diff
@@ -257,6 +257,9 @@ let main () =
     | Errors.Error e ->
       Printer.print_err "%a" Errors.report e;
       exit 1
+    | D_loop.DolmenError (i, descr) ->
+      Printer.print_err "Dolmen failed %s (code %i)" descr i;
```

Comment:
We do not really want to mention Dolmen here because users don't know (or care) what Dolmen is.